### ISSUE TITLE
Add Services to existing Deployment after creation

### ISSUE THREAD

**Issue by naisanza** [kubernetes/dashboard#1569]
##### Environment

```
Dashboard version: gcr.io/google_containers/kubernetes-dashboard-amd64:v1.5.1
Kubernetes Client Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.1", GitCommit:"82450d03cb057bab0950214ef122b67c83fb11df", GitTreeState:"clean", BuildDate:"2016-12-14T00:57:05Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
Kubernetes Server Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.1", GitCommit:"82450d03cb057bab0950214ef122b67c83fb11df", GitTreeState:"clean", BuildDate:"2016-12-14T00:52:01Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
Operating system: Linux k8s-master01 4.8.0-22-generic #24-Ubuntu SMP Sat Oct 8 09:15:00 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
Node.js version:
Go version: 1.7.4
```

##### Steps to reproduce

1. Create new deployment without creating services

##### Observed result

1. No option for adding Service to existing Deployment

##### Expected result

1. Option for adding Service to existing Deployment

**Comment by IanLewis** [comment]
This seems similar to previous requests like #1527

Currently you can create a Service by uploading the YAML definition but don't have a form for creating Services directly yet.

**Comment by naisanza** [comment]
@IanLewis good ol' YAML definitions. How I avoid it so much, hence my excitement when kubernetes/dashboard released

I've done it once before with [Joyent's SmartOS](https://github.com/joyent/smartos-live), which uses `JSON` files for its VM specs. I couldn't remember any of the options for the life of me, and after weeks of referring back, again and again, to the manual on `"what option do I need here?"`, or `"how do I add a disk?"` or `"how do I make it boot from an ISO?"`. I was finally fed up with how much time it took to just to write out one VM spec to use, and not only that, if I wanted something modified I either needed to go through my library of just JSON spec files I've saved and copying portions of what I needed. I always thought, is this really how people use this thing, just memorizing these JSON specs?

So I made a CLI for SmartOS called [create-vm](https://github.com/TheShellLand/sailboat/blob/master/sailboat/opt/custom/bin/create-vm) that lets you go through a small wizard selecting options, or entering a custom option. I've since stopped using the solaris kernel, but for the time being it made life a breeze.

Here are some enhancements that can be done to make using YAML definitions easier:

+ Allow copy and pasting of just the necessary parts of an existing deployment
+ Allow pasting of a new YAML definition when creating a new deployment instead of uploading a file

(Disclaimer: This is after having used Kubernetes for 10 days and some change, avoiding the manuals as much as possible, and glancing over dozens of YAML examples and then thinking to myself "I'll wait for a new update to the dashboard")

Truthfully, I just want to use the system. I like the technology hurdles it overcomes that aligns with what I'm looking for, I love `kubeadm init`, but I just want to get to that point of "this is the best thing ever" since that's what the big shiny words on the front page of kubernetes.io keeps telling me
